### ISSUE TITLE
Reviewer 3.7

### ISSUE THREAD

**Issue by evogytis** [blab/mers-structure#38]
Page 9. I think the section on recombination is verbose, and potentially misleading. You didn't perform an ARG, you performed a perfectly reasonable sensitivity analysis to different partitions of the genome. I would be more upfront about this, and try to explain it more simply and clearly.

**Comment by evogytis** [comment]
Struggling a bit with this, since I wrote the section. Fresh pair of eyes, please?

**Comment by trvrb** [comment]
I believe this is unresolved. Reopening issue and will attempt a textual fix.

**Comment by trvrb** [comment]
Resolved in PR #41.
